File: inventario-backend/src/main/java/cl/uddi/inventario/domain/Producto.java
package cl.uddi.inventario.domain;

public class Producto {
    public int sku;
    public String nombre;
    public String categoria;
    public int precio;
    public int stock;
    public int stockMin;
    public String estado; // ACTIVE | INACTIVE

    public boolean activo() { return "ACTIVE".equalsIgnoreCase(estado); }

    @Override public String toString() {
        return sku+" | "+nombre+" | "+categoria+" | $"+precio+" | stock="+stock+"/"+stockMin+" | "+estado;
    }
}

File: Bootcamp Frontend/Models/Producto.cs
using System.Text.Json.Serialization;

namespace Bootcamp_Frontend.Models
{
    public class Producto
    {
        [JsonPropertyName("sku")]
        public int Sku { get; set; }

        [JsonPropertyName("nombre")]
        public string Nombre { get; set; }

        [JsonPropertyName("stock")]
        public int Stock { get; set; }

        [JsonPropertyName("precio")]
        public double Precio { get; set; }
    }
}
